yt/data_objects/static_output.py

```diff
@@ -150,16 +152,18 @@ class Dataset(abc.ABC):
     coordinates = None
     storage_filename = None
     particle_unions: Optional[Dict[str, ParticleUnion]] = None
-    known_filters = None
+    known_filters: Optional[Dict[str, ParticleFilter]] = None
```

Comment:
So I'm not sure but do you want to try to have this be a `str` or make it explicitly a `ParticleType`?

Reply:
Nice catch, it should be ParticleType !

Reply:
done